src/vs/base/browser/ui/actionbar/actionbar.ts

```diff
@@ -273,7 +277,9 @@ export class ActionItem extends BaseActionItem {
 
 	public _updateLabel(): void {
 		if (this.options.label) {
-			this.$e.text(this.getAction().label);
+			let label = this.getAction().label;
```

Comment:
@sbatten shouldn't the concept of mnemonics only be used when the action shows up in a menu? It seems to me your changes make this happen anywhere the action is showing up. Maybe this should be an option for the action item to pass through to enable this.

Reply:
here I followed the pattern I thought was present in the electron menu. If the action item label has &&, it will create a mnemonic. I only pass these in menu creation. I might be able to move this to ui/menu.ts if you prefer?

Reply:
@bpasero in the interest of getting some insider testing, are you willing to put off changing this in the next check-in? I'd like to focus on items that need to be fixed for initial insider testing so I can then shift to the milestone goal (this included)